### DISCUSSION TITLE
Features

### DISCUSSION THREAD

**Discussion by AndreyDmitriev39r** [AndreyDmitriev39r/random-quote-machine-v2#2] [Ideas]
Write your thoughts on what features application should have

**Comment by MaximusI30** [comment]
I have a few suggestions: 
1. Calm music in the background? 
2. The background picture or color, let's make it more interesting than that? Lets say, a live wallpaper? Maybe nature?
3. Let people change the background? And font as well.
4. Let people save it too?

**Reply by AndreyDmitriev39r** [reply]
1. Yeah, that may be interesting.
2. I am thinking about background image too.
3. I have an idea to change background depending on type of quote. Default quote(for 1st render should be random, background should be generic for that). When quote of certain type is rendered, bg should changerespectively. Making other styling dynamic- fonts or else - also we can consider that. We can do dark mode for example.
4. letting people sve anything means we need backend, so that is for later. For saving prefrences during one session, we can work with localStorage.
